### PR TITLE
Add an example of beamjob and an example of xgboosttrainingjob

### PR DESCRIPTION
**Pull request by xyxxxxx**
* Beam 的示例使用单词计数示例程序
* XGBoost 的示例从 https://github.com/kubeflow/xgboost-operator/tree/master/config/samples/xgboost-dist 修改得到，支持 GPU 训练
* t9kpublic 镜像已上传

### CODE REVIEW

File: job/xgboosttrainingjob/distributed/job.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                - --learning_rate=0.1
+                - --model_path=job/xgboosttrainingjob/distributed/
+              workingDir: /t9k/mnt/tutorial-examples
+              image: t9kpublic/xgboost-1.7.3:sdk-0.5.2
+              ports:
+                - containerPort: 9991
+                  name: xgboost-port
+              resources:
+                limits:
+                  cpu: 100m
```
Comment (ppkube) [inline job/xgboosttrainingjob/distributed/job.yaml:30]: 100m looks incorrect - you are running a compute-intensive ML job

File: job/xgboosttrainingjob/distributed/job.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
```
Comment (ppkube) [inline job/xgboosttrainingjob/distributed/job.yaml:62]: ditto

File: job/xgboosttrainingjob/distributed/job_gpu.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                - --model_path=job/xgboosttrainingjob/distributed/
+                - --use_gpu
+              workingDir: /t9k/mnt/tutorial-examples
+              image: t9kpublic/xgboost-1.7.3:sdk-0.5.2
+              ports:
+                - containerPort: 9991
+                  name: xgboost-port
+              resources:
+                limits:
+                  cpu: 100m
```
Comment (ppkube) [inline job/xgboosttrainingjob/distributed/job_gpu.yaml:31]: ditto

File: job/xgboosttrainingjob/distributed/job_gpu.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
```
Comment (ppkube) [inline job/xgboosttrainingjob/distributed/job_gpu.yaml:65]: ditto

File: job/xgboosttrainingjob/distributed/job_gpu_t9ksched.yaml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                - --model_path=job/xgboosttrainingjob/distributed/
+                - --use_gpu
+              workingDir: /t9k/mnt/tutorial-examples
+              image: t9kpublic/xgboost-1.7.3:sdk-0.5.2
+              ports:
+                - containerPort: 9991
+                  name: xgboost-port
+              resources:
+                limits:
+                  cpu: 100m
```
Comment (ppkube) [inline job/xgboosttrainingjob/distributed/job_gpu_t9ksched.yaml:69]: ditto

**Review by ppkube — CHANGES_REQUESTED** [review]

File: job/xgboosttrainingjob/distributed/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+* `job_gpu_t9ksched.yaml`：使用 GPU 训练，并且由 T9k 调度器进行调度（默认的队列名称为 `default`，在 `spec.scheduler.t9kScheduler.queue` 字段（第 8 行）进行修改）
+
+```shell
+# cd into current directory
+cd ~/tutorial-examples/job/xgboosttrainingjob/distributed/
+# choose one of the following:
+# 1. CPU training
+kubectl create -f job.yaml
+# 2. GPU training
+kubectl create -f job_gpu.yaml
```
[the remark this answers is not in the document]
Comment (xyxxxxx) [reply job/xgboosttrainingjob/distributed/README.md:18]: 实测支持 GPU

File: job/xgboosttrainingjob/distributed/job.yaml (hunk 1 above)
Comment (xyxxxxx) [reply job/xgboosttrainingjob/distributed/job.yaml]: 全部改为 `cpu: 1, memory: 1Gi`

这个示例计算量太小，差不多1s跑完

**Review by ppkube — APPROVED** [review]
lgtm

**Pull request merged**